templates/jenkins/files/README.md

```diff
@@ -1,19 +1,37 @@
-# Jenkins template for Platform.sh
+# Jenkins for Platform.sh
 
-This project provides a starter kit for Jenkins projects hosted on [Platform.sh](http://platform.sh).
+This project provides a starter kit for Jenkins projects hosted on Platform.sh.
 
-We include recommended .platform.app.yaml and .platform files that should suffice for most use cases. You are free to tweak them as needed for your particular site.
+Jenkins is an open source automation server written in Java. Jenkins helps to automate the non-human part of the software development process, with continuous integration and facilitating technical aspects of continuous delivery.
 
-## Platform.sh configuration files
+## Services
+
+* Java 8
+
+## Post-install
+
+1. After running a Jenkins instance by Platform.sh, the post-installation setup wizard begins.
+2. When you first access a new Jenkins instance, you are asked to unlock it using an automatically-generated password.
+3. Browse to the Jenkins service link and wait until the Unlock Jenkins page appears.
+4. From the Jenkins console log output, copy the automatically-generated alphanumeric password 
+5. Access the Jenkins instance with the (ssh command)[https://docs.platform.sh/development/access-site.html#accessing-the-application-with-ssh]
```

Comment:
Er.  This says there's not one but two passwords to go through?  That's not what I saw when I was testing the template.  I just saw the "go read this file from disk" message.

We can probably collapse several of these steps into a single command: `platform ssh cat /app/.jenkins/secrets/initialAdminPassword`.  (With a link to the CLI docs page where appropriate.)

Reply:
easier, fixed. Thank you